server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/DomainsRoutesTest.java

```diff
@@ -255,6 +261,197 @@ void getDomainShouldReturnNotFoundWhenTheDomainIsAbsent() {
                 .statusCode(HttpStatus.NOT_FOUND_404);
         }
 
+        @Nested
+        class DomainAlias{
+            @Test
+            void getAliasesShouldReturnNotFoundWhenDomainDoesNotExist() {
+                when()
+                    .get(DOMAIN + "/aliases")
+                .then()
+                    .contentType(ContentType.JSON)
+                    .statusCode(HttpStatus.NOT_FOUND_404)
+                    .body("statusCode", is(HttpStatus.NOT_FOUND_404))
+                    .body("type", is("InvalidArgument"))
+                    .body("message", is("The domain list does not contain: domain"));
+            }
+
+            @Test
+            void getAliasesShouldReturnEmptyWhenNone() {
+                with().put(DOMAIN);
+
+                when()
+                    .get(DOMAIN + "/aliases")
+                .then()
+                    .contentType(ContentType.JSON)
+                    .statusCode(HttpStatus.OK_200)
+                    .body(".", hasSize(0));
+            }
+
+            @Test
+            void getAliasesShouldReturnCreatedAliases() {
+                with().put(DOMAIN);
+                with().put(ALIAS_DOMAIN);
+                with().put(ALIAS_DOMAIN_2);
+
+                with().put(DOMAIN + "/aliases/" + ALIAS_DOMAIN);
+                with().put(DOMAIN + "/aliases/" + ALIAS_DOMAIN_2);
+
+                when()
+                    .get(DOMAIN + "/aliases")
+                .then()
+                    .contentType(ContentType.JSON)
+                    .statusCode(HttpStatus.OK_200)
+                    .body("source", containsInAnyOrder(ALIAS_DOMAIN, ALIAS_DOMAIN_2));
```

Comment:
if you replace "source" field name by "alias", I think it will be easier to understand

Reply:
Actually I want to have both terms to make the feature easier to understand: "an alias is the source of a domain mapping"